pl-l2d43c1/pl-l2d43c1.xml

```diff
@@ -276,47 +276,86 @@
-          poenitentiam adducit. Paganus est hodie, 
+          poenitentiam adducit. Paganus est hodie,
+          <quote>
+        <cit> 
+        <ref>
           <name>Iudaeus<name>
+        <ref>
           , infidelis est hodie, haereticus est
           hodie, schismaticus est hodie, quid si cras amplectetur catholicam pacem et sequatur
-          catholicam veritatem? Quid si isti, quos in quoeumque genere erroris notas et tanquam
-          desperatissimos damnas, antequam tiniant istamvitam, agunt b poenitentiam, et inveniunt
-          veram 0 vitam in futuro? Nolite ergo ante tempus iudicare quidquam. W. </p>
-        <p xml:id="pl-l2d43c1-d1e147"> Ex his ostenditur, pro singulis peccatoribus in hac vita esse orandum, nec de'aliquo
-          esse ditïidendum, quia converti potest, dum in hac vitaest; quia non potest sciri de
+          catholicam veritatem? Quid si isti, quos in quocumque genere erroris notas et tanquam
+          desperatissimos damnas, antequam finiant istam vitam, agunt poenitentiam, et inveniunt
+          veram vitam in futuro? 
+          <cit>
+            <quote>
+          Nolite ergo ante tempus iudicare quidquam.
+            <quote> 
+          <cit>
+          </p>
+        <p xml:id="pl-l2d43c1-d1e147"> Ex his ostenditur, pro singulis peccatoribus in hac vita esse orandum, nec de aliquo
+          esse diffidendum, quia converti potest, dum in hac vita est; quia non potest sciri de
           aliquo, utrum peccaverit ad mortem vel in-Spiritum sanctum, nisi cum ab hac vita
-          discesserit, nisi forte-alicui per Spiritum sanctum mirabiliter revelatum fuerit. </p>
-        <p xml:id="pl-l2d43c1-d1e150"> Ex praedictis aliquatenus capi» potest, quomodo accipiatur peccatum in Spiritum sanctum,
+          discesserit, nisi forte alicui per Spiritum sanctum mirabiliter revelatum fuerit. </p>
+
+        <p xml:id="pl-l2d43c1-d1e150"> Ex praedictis aliquatenus capi potest, quomodo accipiatur peccatum in Spiritum sanctum,
           scilicet invidentia gratiae fraternitatem impoenitenter oppugnans, quae utique obstinatio
-          esse videtur, et , omnis 'impoenitentis obstinatio atque desperatio. Notandum vero est,
+          esse videtur, et omnis impoenitentis obstinatio atque desperatio. Notandum vero est,
           quod non omnis qui non poenitet, impoenitens dici potest, quia impoenitentia proprie
           obstinati est, et ut quidam volunt, etiam desperati. </p>
-        <p xml:id="pl-l2d43c1-d1e153"> De hoc quoque e peccato in Spiritum sanctum * Ambrosius in libro De Spiritu'ïsauclo 3
-          disserens, delinitam assignationem tra- dit dicens: «Cur 'Dominus dixerit4: Qui
-          blasphemaverit in Filium hominis f remittetur. ei; qui autem blasphemaverit in Spiritum
+
+        <p xml:id="pl-l2d43c1-d1e153"> De hoc quoque peccato in Spiritum sanctum 
+        <ref>
+          <name>Ambrosius 
```

Comment:
`<name>` here is missing a closing tag
(and generally all of your closing tags are missing the slash)